### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by PiyushNegi1703**
Created a class and a dummy object for a user

### CODE REVIEW

**Comment by th3c0d3br34ker** [comment]
hey @PiyushNegi1703, please add a loom video along with the explanation.

Refer: [Link 1](https://wiki.kalvium.community/s/c5e72797-4de2-484b-839b-8a7fc7a8a03d#h-how-to-request-a-review-for-integrity-checks) | [Link 2](https://wiki.kalvium.community/s/3ba46b3d-824e-46ad-98ec-b9e07ba0decb)

**Comment by PiyushNegi1703** [comment]
Hi @th3c0d3br34ker, here is the Loom video for my PR
https://www.loom.com/share/1c6d823e273247fb9ba14cd5fbf9b7b6?sid=df9c10fd-8682-4237-8168-eac92639575a

File: classes.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class User:
+    def __init__(self, name: str, email: str, password: str):
+        self.name = name
+        self.email = email
+        self.password = password
```
Comment (th3c0d3br34ker) [inline classes.py:5]: Consider using hashing and then storing the passwords.

**Review by th3c0d3br34ker — APPROVED** [review]
thanks @PiyushNegi1703, i reviewed your submission.

Overall comments:

1. Please add all the remaining classes mentioned in the proposal, failing to add them in future may result in reduction of marks.
2. This PR is satisfactory for the concept, but lacks over all complexity. 

you may merge is PR.

File: classes.py (hunk 1 above)
Comment (PiyushNegi1703) [reply classes.py:5]: Yes, you will be able to see all the functionalities and better complexity of the code in the upcoming PRs.

**Pull request merged**